bot.py

```diff
@@ -338,6 +338,147 @@ async def display_current_list_of_summoners(ctx):
+            if len(summoner_to_remove_input) > 0:
+                raise Exception(
+                    "Unregistered Summoner(s)",
+                    "Summoners: {0} were not registered for the game".format(
+                        str(summoner_to_remove_input)
+                    )
+                )
+        else:
+            raise Exception(
+                "Limit Exceeded",
+                "There is no summoner(s) added in the game.\nPlease add summoner(s) first!"
+            )
+        # display list of summoners
+        await display_current_list_of_summoners(ctx)
+
+    # pylint: disable=broad-except
+    except Exception as e_values:
+        if "404" in str(e_values):
+            error_title = "Invalid Summoner Name"
+            error_description = f"`{e_values.args[1]}` is not a valid name. \
+                \n\nAdding multiple summoners:\n `@{bot.user.name} add name1, name2`"
+        elif "Limit Exceeded" in str(e_values) or "Unregistered Summoner(s)" in str(e_values):
+            error_title = e_values.args[0]
+            error_description = e_values.args[1]
+        else:
+            error_title = f"{e_values}"
+            error_description = "Oops! Something went wrong.\nTry again!"
+
+        embed_data = EmbedData()
+        embed_data.title = ":x:   {0}".format(error_title)
+        embed_data.description = "{0}".format(error_description)
+        embed_data.color = discord.Color.red()
+        await ctx.send(embed=create_embed(embed_data))
+
+        # display list of summoners
+        await display_current_list_of_summoners(ctx)
+
+
+def normalize_name(string):
+    """normalize name by changing to lower case and removing whitespaces"""
+    return string.lower().replace(" ","")
+
+
+# pylint: disable=too-many-locals, too-many-branches, too-many-statements
+@bot.command(name="clear", help="Clear player(s) from the list")
+async def clear_list_of_summoners(ctx):
+    """Clear out summoners from the list"""
+
+    try:
+        # for importing data from json file
+        file_data = ""
+        # initializing server id to a variable
+        server_id = str(ctx.guild.id)
+
+        if os.path.getsize(json_path) > 0:
+            with open(json_path, "r") as file:
+                file_data = json.load(file)
+
+        if server_id in file_data.keys():
+            file_data[server_id].clear()
+            with open(json_path, "w") as file:
+                json.dump(file_data, file, indent=4)
+
+        # display list of summoners
+        await display_current_list_of_summoners(ctx)
+
+    # pylint: disable=broad-except
+    except Exception as e_values:
+        error_title = f"{e_values}"
+        error_description = "Oops! Something went wrong.\nTry again!"
+
+        embed_data = EmbedData()
+        embed_data.title = ":x:   {0}".format(error_title)
+        embed_data.description = "{0}".format(error_description)
+        embed_data.color = discord.Color.red()
```

Comment:
Are we going to have similar embed formed for these cases?
Maybe makes sense to factor this out to `createGeneralErrorEmbed()` function

Reply:
This issue will be handled separately from this PR with [Issue#39](https://github.com/dhmoon91/discord/issues/39)